src/qnngds/devices/nanowire.py

```diff
@@ -22,16 +26,17 @@ def spot(
         Device: A device containing 2 optimal steps joined at their channel_w end.
     """
 
-    NANOWIRE = Device(f"NANOWIRE {channel_w} ")
+    NANOWIRE = Device()
```

Comment:
yes, its a detail. This is because some devices are renamed at the end of the functions. But in any case NO device is returned from any function without a clean name now. Maybe if you were to check in the middle of the functions the device's name it would not be correct are would even be unnamed, but that is more about cleaning the code than an actual issue. Good point though, for clarity it would be good to do.